src/polar/Pie.tsx

```diff
@@ -112,7 +113,7 @@ interface InternalPieProps extends PieDef {
   hide?: boolean;
   /** the input data */
   data?: any[];
-  sectors?: PieSectorDataItem[];
+  sectors?: Readonly<PieSectorDataItem[]>;
```

Comment:
ah, I thought they were the same. it should be `ReadonlyArray<T>`. My version cannot be assigned to but it can be mutated with methods, the latter cant do either